scripts/deployment/main.py

```diff
@@ -158,6 +159,30 @@ def deploy_beamer(
         )
     )
 
+    tokens = l2_config.get("token", [])
+
+    for token_arguments in tokens:
+        token_address = token_arguments["token_address"]
+        if token_address == "deploy_mintable_token":
+            if token is None:
+                raise ValueError(
+                    "Expecting mintable token to be deployed. (option --deploy-mintable-token)"
+                )
+            token_address = token.address
+        decimals = (
+            web3.eth.contract(address=token_address, abi=CONTRACTS["MintableToken"][0])
```

Comment:
We could move `beamer.chain._load_ERC20_abi` to `beamer.util`, make it public and use it here.

Reply:
good idea